src/main/java/com/example/bookshop/controller/BookController.java

```diff
@@ -21,48 +25,58 @@
 import org.springframework.web.bind.annotation.ResponseStatus;
 import org.springframework.web.bind.annotation.RestController;
 
+@Tag(name = "Book management", description = "Endpoints for managing books")
 @Validated
 @RequiredArgsConstructor
 @RestController
 @RequestMapping("/books")
 public class BookController {
     private final BookService bookService;
 
-    @GetMapping("/author")
-    public List<BookDto> findAllByAuthor(@RequestParam String author) {
-        return bookService.findAllByAuthor(author);
+    @Operation(summary = "find all books where authors with certain name",
+                description = "find all books by authors")
+    @GetMapping("/{author}")
+    public List<BookDto> findAllByAuthor(@PathVariable String author,
+                                         @PageableDefault Pageable pageable) {

```

Comment:
you may also add `@ParameterObject` annotation for your pageable

this will make it look better on Swagger